### PR TITLE
Guest OS customizations for photon

### PR DESCRIPTION
**Pull request by kaushik229**

### CODE REVIEW

File: SPECS/open-vm-tools/open-vm-tools.spec [hunk 1]
```diff
@@ -1,15 +1,17 @@
 Summary:	Usermode tools for VmWare virts
 Name:		open-vm-tools
 Version:	9.10.0
-Release:	2%{?dist}
+Release:	3
```
Comment (toliaqat) [inline SPECS/open-vm-tools/open-vm-tools.spec]: Make this 3%{?dist}

File: SPECS/open-vm-tools/open-vm-tools.spec [hunk 2]
```diff
@@ -58,7 +64,6 @@ EOF
 make DESTDIR=%{buildroot} install
 rm -f %{buildroot}/sbin/mount.vmhgfs
 %post
-ln -s /usr/sbin/mount.vmhgfs /sbin/mount.vmhgfs
```
Comment (sharathjg) [inline SPECS/open-vm-tools/open-vm-tools.spec]: why did you remove this?
